### DISCUSSION TITLE
Simbrief Route cross the Atlantic

### DISCUSSION THREAD

**Discussion by FGrabi** [flybywiresim/aircraft#3373] [Q&A]
Hello,

I need some help on flying the following route from Simbrief (LPAZ-EINN IVAO-Worldtour):
XUVA1N XUVAP DCT BAVAS DCT 42N022W DCT 45N020W DCT KOGAD DCT ATSUR DCT BUNON BUNO2E

During the INIT in MCDU I see messages that some waypoint are "Not in Database". Waypoints are 42N022W and 45N020W. After passing BAVAS the aircraft goes directly to KOGAD. If I check the route in STKP on the Live Map the missing waypoints are shown with the "normal" route.
Question: What do I need to do to fly the exact route (after BAVAS)? Any help is appreciated. Thx.

Greets
Frank

**Comment by Inaxair** [comment]
Hello,

this might be a problem with the database? Like AIRAC valid period...

Best regards,
Inaxair from FlyByWire Simulations

**Reply by FGrabi** [reply]
That was also my idea. Simbrief uses the actual 2101 circle and I updated the MSFS with the actual circle (via Navigraph) lso with 2101. The MCDU showing still the former cycle (valid until 27Jan). Opened a Bug request a few days ago (#3233). So if the answers from the FBW Team is correct (as always) this is only a wrong info in MCDU because MS or ASOBO or both are responsible for the info in MCDU. The plane will use the loaded 2101.

#3233

**Reply by FGrabi** [reply]
Just see in the ISSUE Section, that I'm not the only one with the problem. See #3385
